SlowCheetah/Properties/AssemblyInfo.cs

```diff
@@ -25,6 +25,9 @@
 // The following GUID is for the ID of the typelib if this project is exposed to COM
 [assembly: Guid("6354d859-e629-49fc-b154-fc0ba42d71b0")]
 
+// Allow the test to see the internals of this class
+[assembly: InternalsVisibleTo("SlowCheetah.Tests")]
```

Comment:
That is a very good point. Currently the only reason the internals are visible to the tests are for testing some utility methods. I don't think these methods should be public, but at the same time, I think it's good to test them so they don't hinder the API tests.